src/raftstore/store/peer.rs

```diff
@@ -248,7 +248,7 @@ impl Peer {
             pending_cmds: Default::default(),
             peer_cache: store.peer_cache(),
             coprocessor_host: CoprocessorHost::new(),
-            size_diff_hint: 0,
+            size_diff_hint: cfg.region_check_size_diff,
```

Comment:
here means all the region will be checked when the store starts, is it necessary?